deeplinkdispatch-processor/src/main/java/com/airbnb/deeplinkdispatch/DeepLinkProcessor.java

```diff
@@ -116,13 +113,14 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
         = !roundEnv.getElementsAnnotatedWith(DeepLinkHandler.class).isEmpty();
 
     if (!deepLinkElements.isEmpty()) {
+      String packageName = deepLinkElements.get(0).getPackageName();
```

Comment:
yes, if you annotate classes in different packages, we'll pretty much use the package from the first one. Since I don't think order is guaranteed, we'll effectively pick a random package to use, based on all of the packages of the annotated classes 😬 
If you can think of a better solution plz let me know